resources/deployments/controller/controller_deployments.go

```diff
@@ -173,8 +173,12 @@ func (d *DeploymentsController) AbortDeployment(w rest.ResponseWriter, r *rest.R
 	d.view.RenderEmptySuccessResponse(w)
 }
 
+const (
+	GetDeploymentForDeviceQueryArtifact   = "artifact_name"
+	GetDeploymentForDeviceQueryDeviceType = "device_type"
```

Comment:
If you are making it here, why not include the attribute `device_type` also in this PR?